root/static/html/help.html

```diff
@@ -139,8 +139,8 @@ <h4>Can you guarantee a user will submit a pull request to my repository?</h4>
   <br>
 
 
-  <h4>How can I know who have received my repositories as their assignments?</h4>
   <p>You can see your assignees in <a href="/history">history</a> page.
+  <h4>How can I know who has received my repositories as their assignments?</h4>
```

Comment:
Hi! Was this line made by mistake by any chance? It seems the question now shows up after the answer.

Reply:
Hrm, that's weird. Sorry about that!